functionnal_tests/cypress/integration/helpers/htmldoc.js

```diff
@@ -0,0 +1,27 @@
+import { assertPopupCreateContent } from './index.js'
+
+// INFO - CH - 2019-05-15 - The function bellow assumes you already is on a workspace's content list page
+export const create_htmldocument = (cy, htmlDocTitle = 'Html document 1') => {
```

Comment:
use createHtmlDocument from `db_commands` instead

Reply:
I wont refactor it now, this branch needs to be fixed.
I'll just change the comments to tell to use the one from db_commands